hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreScanner.java

```diff
@@ -728,7 +739,7 @@ public boolean next(List<Cell> outResult, ScannerContext scannerContext) throws
             break;
 
           case SEEK_NEXT_COL:
-            seekOrSkipToNextColumn(cell);
+            doSeekCol(cell);
```

Comment:
I thought it is not need to change this method name? You can add the new logic to seekOrSkipToNextColumn method directly?

Reply:
Ok . I can see how can that be done.